tests/test_project/models.py

```diff
@@ -30,3 +30,135 @@ def __str__(self):
+                                          db_column='acctinputoctets',
+                                          null=True,
+                                          blank=True)
+    output_octets = models.BigIntegerField(verbose_name=_('output octets'),
+                                           db_column='acctoutputoctets',
+                                           null=True,
+                                           blank=True)
+    called_station_id = models.CharField(verbose_name=_('called station ID'),
+                                         max_length=50,
+                                         db_column='calledstationid',
+                                         db_index=True,
+                                         blank=True,
+                                         null=True)
+    calling_station_id = models.CharField(verbose_name=_('calling station ID'),
+                                          max_length=50,
+                                          db_column='callingstationid',
+                                          db_index=True,
+                                          blank=True,
+                                          null=True)
+    terminate_cause = models.CharField(verbose_name=_('termination cause'),
+                                       max_length=32,
+                                       db_column='acctterminatecause',
+                                       blank=True,
+                                       null=True)
+    service_type = models.CharField(verbose_name=_('service type'),
+                                    max_length=32,
+                                    db_column='servicetype',
+                                    null=True,
+                                    blank=True)
+    framed_protocol = models.CharField(verbose_name=_('framed protocol'),
+                                       max_length=32,
+                                       db_column='framedprotocol',
+                                       null=True,
+                                       blank=True)
+    framed_ip_address = models.GenericIPAddressField(verbose_name=_('framed IP address'),
+                                                     db_column='framedipaddress',
+                                                     db_index=True,
+                                                     # the default MySQL freeradius schema defines
+                                                     # this as NOT NULL but defaulting to empty string
+                                                     # but that wouldn't work on PostgreSQL
+                                                     null=True,
+                                                     blank=True)
+
+    def save(self, *args, **kwargs):
+        if not self.start_time:
+            self.start_time = now()
+        super(AbstractRadiusAccounting, self).save(*args, **kwargs)
```

Comment:
Know it too I sent PR because in my local machine I was getting weird error:
```
from openwisp_utils.admin import (MultitenantAdminMixin, MultitenantOrgFilter,
ImportError: cannot import name 'ReadOnlyAdmin'
```

Reply:
that's probably because of the modifications you have made in your previous task, check that

Reply:
@nemesisdesign For this task I am using unmarged version of repo (without my changes from previous task)